tests/runtest.py

```diff
@@ -32,35 +32,34 @@
 import time
 import subprocess
 
-if len(sys.argv) not in [5, 6, 7, 8, 9, 10]:
-    print("""Usage: %s input_file logfile doperltest top_srcdir doreaptest alt_output_file alt_psi4_exe alt_psi4datadir""" % (sys.argv[0]))
+if len(sys.argv) not in {4, 5, 6, 7, 8, 9}:
+    print("""Usage: %s input_file logfile top_srcdir doreaptest alt_output_file alt_psi4_exe alt_psi4datadir""" % (sys.argv[0]))
     sys.exit(1)
 
 # extract run condition from arguments
 python_exec = sys.argv[0]
 infile = sys.argv[1]
 logfile = sys.argv[2]
-psiautotest = sys.argv[3]
-top_srcdir = sys.argv[4]
-sowreap = sys.argv[5]
+top_srcdir = sys.argv[3]
+sowreap = sys.argv[4]
```

Comment:
sowreap is also dead, btw. but don't mess up the commit history just on that account.

Reply:
I'd rather not add that to this PR. There are additional questions about what to do with the tests currently tagged `sowreap` but not in the CMake. Sounds like a worthy topic for an issue, though. 🙂